experiments/calibration/calibrate_land.jl

```diff
@@ -30,20 +28,27 @@ rng_seed = 2
 rng = Random.MersenneTwister(rng_seed)
 FT = Float64
 
-include(joinpath(@__DIR__, string("priors_", model_type, ".jl")))
+include(
+    joinpath(
+        pkgdir(ClimaLand),
+        "experiments",
+        "calibration",
+        string("priors_", model_type, ".jl"),
+    ),
+)
 # potentially we could add loss-parameters pairing https://clima.github.io/EnsembleKalmanProcesses.jl/dev/update_groups/
 
 ekp_process = EKP.Unscented(prior)
 ensemble_size = ekp_process.N_ens
 
 # Config for workers
-CAL.add_workers(
-    ensemble_size;
-    cluster = :auto,
-    device = :gpu,
-    l_walltime = "11:30:00", # this only works for PBS. ClimaCalibrate will soon have a new more generalizable keyword for it to work on Slurm and PBS.
-    l_job_priority = "premium", # this only works on PBS
-)
+addprocs(CAL.SlurmManager())
```

Comment:
Should this be set based on a flag at the top (e.g. job_manager = "PBS" vs "slurm"?